tests/Unit/Domain/Creators/Test_CylindricalBinaryCompactObject.cpp

```diff
@@ -291,11 +291,13 @@ void test_connectivity_once(const bool with_sphere_e,
     const auto& excision_sphere_a =
         domain.excision_spheres().at("ObjectAExcisionSphere");
     CHECK(excision_sphere_a.radius() == inner_radius_objectA);
-    CHECK(excision_sphere_a.center() == center_objectA);
     const auto& excision_sphere_b =
         domain.excision_spheres().at("ObjectBExcisionSphere");
     CHECK(excision_sphere_b.radius() == inner_radius_objectB);
-    CHECK(excision_sphere_b.center() == center_objectB);
+    for (size_t i = 0; i < 3; ++i) {
```

Comment:
If it's possible to do this with `CHECK_ITERABLE_APPROX` I would do that, otherwise this is ok

Reply:
I think this check should be exact because the floats get copied at most.